src/aspire/orientation/__init__.py

```diff
@@ -1,38 +1,497 @@
 import logging
+import math
+import numpy as np
+import scipy.sparse as sparse
+
+from aspire import config
+
+from aspire.basis.polar_2d import PolarBasis2D
+from aspire.source import ArrayImageSource
+from aspire.utils.coor_trans import common_line_from_rots
+from aspire.utils.matlab_compat import m_reshape
 
 logger = logging.getLogger(__name__)
 
 
-class Orient3D:
+class CLOrient3D:
     """
-    Define a base class for estimating 3D orientations
+    Define a base class for estimating 3D orientations using common lines methods
     """
-    def __init__(self, src):
+    def __init__(self, src, n_rad=None, n_theta=None, n_check=None):
         """
-        constructor of an object for estimating 3D orientations
+        Initialize an object for estimating 3D orientations using common lines
+
+        :param src: The source object of 2D denoised or class-averaged imag
+        :param n_rad: The number of points in the radial direction
+        :param n_theta: The number of points in the theta direction
+        :param n_check: For each image/projection find its common-lines with
+            n_check images. If n_check is less than the total number of images,
+            a random subset of n_check images is used.
         """
-        pass
+        self.src = src
+        self.n_img = src.n
+        self.n_res = self.src.L
+        self.n_rad = n_rad
+        self.n_theta = n_theta
+        self.n_check = n_check
+        self.clmatrix = None
+        self.shifts_1d = None
```

Comment:
Looks like this is no longer used. Remove.

Reply:
Done in commit e1c9992.